packages/gatsby-plugin-gatsby-cloud/src/gatsby-node.js

```diff
@@ -23,6 +23,21 @@ exports.onCreateWebpackConfig = ({ actions, stage }) => {
         merge: true,
       }),
     ],
+    module: {
+      rules: [
+        {
+          test: /\.(png|jp(e*)g|svg|gif)$/,
+          use: [
+            {
+              loader: `file-loader`,
+              options: {
+                name: `images/[hash]-[name].[ext]`,
+              },
+            },
+          ],
+        },
+      ],
+    },
```

Comment:
I don't think we need this as icons are part of JSX. This would also be dangerous as you're changing it for the public build too

Reply:
Right, I think this from when I was trying to figure out how to import svgs. I can get rid of that